### PR TITLE
[WIP] [MIG] auditlog: Migration to 13.0

### PR DESCRIPTION
**Pull request by lembregtse**
Currently having a problem (with what I guess is an Odoo 12.5/13 bug) where the computed Many2many on ir.models "inherited_model_ids" is causing a CacheMis exception.

### CODE REVIEW

**Comment by sebalix** [comment]
@lembregtse can you cherry-pick the commits in https://github.com/OCA/server-tools/pull/1688? Not sure it'll fix the issue with `CacheMiss` however :)

**Comment by sebalix** [comment]
@lembregtse the `CacheMiss` issue on Travis is related to this: https://github.com/odoo/odoo/pull/37957
Tests are running fine with this fix.

**Comment by sebalix** [comment]
@lembregtse odoo/odoo#37957 has been merged, tests regarding `auditlog` are now OK, failing jobs are related to `database_cleanup` which isn't migrated yet, you should comment lines here: https://github.com/OCA/server-tools/blob/13.0/.travis.yml#L21 (all lines with `INCLUDE=database_cleanup`).
Can you also cherry-pick commits from #1688 ? If you've not the time I'll do it myself once your PR is merged.

**Comment by lembregtse** [comment]
@sebalix I will do so tomorrow, I have been traveling remote so I had limited time. Thanks for your research and fixes!

File: auditlog/models/log.py
Comment (dreispt) [inline auditlog/models/log.py:15]: Maybe we could also get rid of the `size`. Any good reason to keep it?

**Review by dreispt — APPROVED** [review]
LGTM
Good to merge after resolving conflicts in `.isort.cfg`

File: auditlog/models/log.py
Comment (sebalix) [reply auditlog/models/log.py:15]: No good reason to me.

**Comment by sebalix** [comment]
I copied this PR to https://github.com/OCA/server-tools/pull/1793 + added some other fixes from 11.0/12.0 branches. This PR can be closed.

**Pull request closed without merging**
